04-Enumerable/Enumerable/Task.cs

```diff
@@ -239,9 +272,10 @@ public class Task {
         ///   { "aaa", "aaa", "aaa" } =>  { ("aaa", 3) } 
         ///   { } => { }
         /// </example>
-        public IEnumerable<Tuple<string,int>> GetCountOfStrings(IEnumerable<string> data) {
+        public IEnumerable<Tuple<string, int>> GetCountOfStrings(IEnumerable<string> data)
+        {
             // TODO : Implement GetCountOfStrings
-            throw new NotImplementedException();
+            return data.Select(item => Tuple.Create(item, data.Where(result => result == item).Count())).Distinct();
```

Comment:
condition can be moved to count
also please split it by lines, bit hard to read